trust-quorum/src/coordinator_state.rs

```diff
@@ -24,9 +25,11 @@ use std::time::Instant;
 /// allows progress to always be made with a full linearization of epochs.
 ///
 /// We allow some unused fields before we complete the coordination code
-#[allow(unused)]
 pub struct CoordinatorState {
+    log: Logger,
+
     /// When the reconfiguration started
+    #[allow(unused)]
```

Comment:
```suggestion
    #[expect(unused)]
```

Presumably this is useful for debugging.

Reply:
I didn't even know about `expect` until your last PR. Thanks!